src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -5,6 +5,7 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_ATTENDANCE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_BIRTHDAY_DATE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MATRICULATIONYEAR;
```

Comment:
Would it be too much work to make it PREFIX_MATRICULATION_YEAR? In order to follow the existing format of having each word separated with an underscore.

Or is there any reason you chose not to separate matriculation and year?

Reply:
Thanks for the suggestion! I have updated the code accordingly with the suggested changes.